site/layouts/course/single.html

```diff
@@ -121,7 +121,10 @@ <h2>Why Learn With NKD Agility?</h2>
       <div class="col-xl-6"></div>
     </div>
   </section>
-  {{- if and (not (eq .Page.Params.syllabus nil)) (gt (len .Page.Params.syllabus) 0) }}
+  {{/* Get syllabus data from file or front matter */}}
```

Comment:
This is unessesary as its duplicated in `site/layouts/partials/delivery-parts/syllabus-single.html`. Remvoe from here